File: JAVA/MyHelloJava03/src/javastudy14_1012/Student.java
package javastudy14_1012;

public class Student {
	private String name;
	private int age;
	public Student(String name, int age) {
		super();
		this.name = name;
		this.age = age;
	}

	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public int getAge() {
		return age;
	}
	public void setAge(int age) {
		this.age = age;
	}

	//alt shift s  + generate ToString. 
	//Object 클래스의 toString 메소드를 오버라이딩함
	@Override
	public String toString() {
		//return super.toString();
		return "Student [name=" + name + ", age=" + age + "]";
	}


}

File: JS/0929/student.js
class Student{  //클래스 선언
    constructor(name,gender,beonho)
    {
        this.name=name
        this.gender=gender
        this.beonho=beonho
    }
    // toString(){Student.prototype.toString=function(){  
    //     return `${this.name} 학생은 ${this.gender}학생이며, `+
    //     `${this.beonho}번이다.`
    // }}   >>메모리 낭비라 바깥으로 뺐다.
}

//toString 함수 정의
Student.prototype.toString=function(){  
    return `${this.name} 학생은 ${this.gender}학생이며, `+
    `${this.beonho}번이다.`
}

//배열 선언
let student = [] 
